torch_glow/tests/utils.py

```diff
@@ -1,17 +1,21 @@
+from argparse import Namespace
+
 import torch
 import torch_glow
 
 GLOW_NODE_NAME = "glow::CompilationGroup"
+SUBGRAPH_ATTR = "Subgraph"
+OP_KIND = Namespace(Add="aten::add", AddInplace="aten::add_")
```

Comment:
I don't think it exists for all nodes, the point is that these are supposed to be registerable. I don't think this is really needed, it would be fine to have `expected_fused_ops=["aten::add"]`

Reply:
I'm just worried that we'll repeat `aten::add` (and similar strings) in a bunch of places. The add tests themselves have regular, inplace and broadcasting tests, and add is used in other operator tests to make sure there is more than one op for PT to fuse.

Reply:
>I'm just worried that we'll repeat aten::add (and similar strings) in a bunch of places.

I still think having "aten::add" repeated a bunch of times is better than adding this random other Namespace stuff (imo).

>add is used in other operator tests to make sure there is more than one op for PT to fuse."

Thats a good point, I think we should check that all of things that should be in the subgraph are in the subgraph and that nothing that should be in the subgraph is not in the subgraph but not check that other things are not in the subgraph. what do you think?

Reply:
Sorry meant to say that also other things can be in the subgraph even if not in expected_fused_ops

Reply:
Okay, let's check that all ops in `expected_fused_ops` are found inside the Glow compilation group and not outside. The test should not fail if any ops that are *not* in that list can appear inside or outside of the group.